### ISSUE TITLE
anomaly with omit and forced scenes (and initially one more thing but I seem to have been wrong)

### ISSUE THREAD

**Issue by ftolsson** [lmparppei/Beat#61]
So here's two very small issues that have combined into a slightly larger one for me. 

First: 

It seems for any scene starting with INT or EXT, an omission can go directly before like this:
```
/*
INT. SOMEWHERE
```
and the scene will still know that it is a scene, and be correctly flagged as omitted.

But a forced scene, like this:
```
/*
.LATER IN THE SAME PLACE
```
the scene will forget that it is a scene, and just become a line of hidden text. For the forced scene to stay aware that it is a scene, there must be a blank line between the `/*` and the `.SCENE`:
```
/*

.LATER IN THE SAME PLACE
```

Not a biggie in itself of course, just add the blank line. But wanting to add omitting/un-omitting from inside my plugin, this inconsistency coincides with another, also very small, issue: 
It seems that Beat does not allow me to add two `newlines` in direct sequence (`Beat.addString("/*\n\n", position)` -- one will be ignored no matter how I try to do it. It even holds true if I add them in two separate `addString` commands -- I simply can't programmatically add two newlines after each other. Putting spaces between doesn't help either.

The result is that I can't correctly omit a forced scene — because the forced scene demands the two newlines that Beat won't allow me to insert. Can this be remedied? (Ideally by making forced scenes not become action lines following an omission, but allowing deliberate double newlines would be a great second choice...)

**Comment by ftolsson** [comment]
Or wait -- once again, I called bug too quickly. In fact, it seems like I *can* indeed add two consecutive newlines — there are just circumstances when it doesn't work. I will have to track down what that is — disregard my report for now. 

(Except the anomaly with obligatory blanklines before forced scenes, that part was true.)

**Comment by lmparppei** [comment]
Fixed in `1.89.8`.

This was caused by a conditional that the preceding line had to be empty when forcing a scene heading. It was there because our American friends love their guns, so some people tended to begin dialogue using something like ".44 is enough to blow their brains out".
